port/unix/omrsysinfo.c

```diff
@@ -3380,6 +3384,26 @@ retrieveLinuxMemoryStatsFromProcFS(struct OMRPortLibrary *portLibrary, struct J9
 		} /* end if else-if */
 	} /* end while() */
 
+	swappinessFs = fopen(PROC_SYS_VM_SWAPPINESS, "r");
+	if (NULL == swappinessFs) {
+		Trc_PRT_retrieveLinuxMemoryStats_failedOpeningSwappinessFs(errno);
+		rc = OMRPORT_ERROR_SYSINFO_ERROR_SWAPPINESS_OPEN_FAILED;
+		goto _cleanup;
+	}
+
+	rc = fscanf(swappinessFs, "%" SCNu64, &memInfo->swappiness);
```

Comment:
Using `rc` for two purposes makes the code confusing: i) to store the error code for `retrieveLinuxMemoryStatsFromProcFS` and ii) to store the return value of `fscanf`. A new local variable with an appropriate name should be used for the return value of `fscanf`.